### PR TITLE
Add react-native-tooliplize library

### PR DESCRIPTION
**Pull request by amandeepmittal**
# 📝 Why & how

Add [react-native-tooltiplize](https://github.com/Stringsaeed/react-native-tooltiplize) which is a new universal tooltip library.

# ✅ Checklist

- [x] Added library to **`react-native-libraries.json`**

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -8864,5 +8864,20 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/Stringsaeed/react-native-tooltiplize",
+    "npmPkg": "https://www.npmjs.com/package/react-native-tooltiplize",
```
Comment (Simek) [inline react-native-libraries.json:8870]:
```suggestion
```

This field only should contain npm package name to validate correctly, but it is not needed when GitHub repository name matches package name.

**Review by Simek — COMMENTED** [review]
Thank you for adding new library to the directory! 👍 

Left a suggestion which should fix the CI, besides that LGTM.

File: react-native-libraries.json (hunk 1 above)
Comment (amandeepmittal) [reply react-native-libraries.json:8870]: Thanks for the update!

**Review by Simek — APPROVED** [review]
🚀

**Pull request merged**